docs/views/pages/Examples/Fade.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import { Fade } from "../../../../src";
+import "../../../app.css";
+
+class FadeExample extends Component {
```

Comment:
Can we make this a functional component also?

Reply:
Sure will do,  I actually did'nt notice this